src/plugins/widgets/linechartwidget.js

```diff
@@ -0,0 +1,146 @@
+var _ = require('lodash');
```

Comment:
Maybe put each widget in a folder so we can have Readme.mds for each of them? Maybe a dedicated folder for each plugin is not necesseary, but we could group the different ChartWidgets in the same folder for instance.